message.proto

```diff
@@ -10,33 +10,33 @@ package mainflux;
 
 // RawMessage represents a message emitted by the Mainflux adapters layer.
 message RawMessage {
-	string Channel     = 1;
-	string Publisher   = 2;
-	string Protocol    = 3;
-	string ContentType = 4;
-	bytes  Payload     = 5;
+	string channel     = 1;
```

Comment:
@dusanb94 just curious, why did you removed capital letters here? Somethign with proto?

Reply:
@manuio There are 2 major reasons, none of which is too much important:
1) There is no need for these fields to be uppercase.
2) Generated JSON tags will be lowercase this way.
Apart from these cosmetic changes, everything will work the same as before.